Assets/Plugins/Source/Editor/Utility/GUIEditorUtility.cs

```diff
@@ -24,12 +24,13 @@ namespace PlayEveryWare.EpicOnlineServices.Editor.Utility
 {
     using System;
     using System.Collections.Generic;
-    using System.Security.Cryptography;
     using UnityEditor;
     using UnityEngine;
 
     public static class GUIEditorUtility
     {
+        private const float MaximumButtonWidth = 100f;
```

Comment:
nit: SCREAMING_SNAKE_CASE

Reply:
I'm split on this. @andrew-hirata-playeveryware Do we have a final answer on our standards for this? I recall making a comment regarding standards being different between C# and C++, but I do not recall what the outcome of that discussion was.